linter.js

```diff
@@ -0,0 +1,42 @@
+// Linter Issue Example
+var myVar = 'Hello';  // Unused variable
+
+function AddTask(taskname) {  // Function name should be camelCase
+    if(taskname == ""){  // Use strict equality (===) instead of loose equality (==)
+        console.log("Task name cannot be empty")
+        return
+    }
+    var taskList = []
+    taskList.push(taskname)
+    taskList.forEach(task => {
+       console.log(task) // Improper indentation
+    })
+}
+
+function deleteTask(id){
+  tasklist.splice(id, 1) // tasklist variable is undeclared (should be taskList)
+}
+
+let sampleTask = {
+    id: 1
+    name: "Sample Task" // Missing comma after object key-value pair
+};
+
+for(let i = 0; i < 10; i++){   // Incorrect spacing around operators
+console.log(i) // Improper indentation
+}
+
+const taskDone = true
+if(taskDone){console.log("Task completed")} // No spaces inside if statement, missing semicolon
+
+let test = 3;
+switch (test){
+  case 1: console.log('One') break // Missing semicolons and proper case formatting
+  case 2: console.log('Two') break
+  case 3: console.log('Three') break
+  default: console.log('Unknown') break // Extra "break" in default case
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Improve switch statement formatting</b></div>
<div id="fix">

The `switch` statement cases are missing semicolons and proper formatting. Consider adding semicolons after each `console.log` statement and formatting each case on separate lines for better readability.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -34,4 +34,16 @@
 -  case 1: console.log('One') break // Missing semicolons and proper case formatting
 -  case 2: console.log('Two') break
 -  case 3: console.log('Three') break
 -  default: console.log('Unknown') break // Extra "break" in default case
 +  case 1:
 +    console.log('One');
 +    break;
 +  case 2:
 +    console.log('Two');
 +    break;
 +  case 3:
 +    console.log('Three');
 +    break;
 +  default:
 +    console.log('Unknown');
 +    // Removed extra "break" in default case

```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/aakashgusain1990/BitoPythonTest/pull/18#issuecomment-2437768865>#77bc9a</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged